test/microbench/org/apache/cassandra/test/microbench/instance/ReadTestSmallPartitionsBench.java

```diff
@@ -39,7 +39,7 @@
 @Fork(value = 1)
 @Threads(1)
 @State(Scope.Benchmark)
-public class ReadTestSmallPartitions extends ReadTest
+public class ReadTestSmallPartitionsBench extends ReadTest
```

Comment:
Nit: I would get rid of `Test` here and in the next file: `ReadSmallPartitionsBench`, `ReadWidePartitionsBench`. Consider also renaming the base class from `ReadTest` to `ReadBench`.